libs/community/langchain_community/graphs/falkordb_graph.py

```diff
@@ -58,18 +58,21 @@ def __init__(
     ) -> None:
         """Create a new FalkorDB graph wrapper instance."""
         try:
-            import redis
-            from redis.commands.graph import Graph
+            from falkordb import FalkorDB
```

Comment:
this is a breaking change — thoughts on falling back to redis if falkordb isn't installed and raising a warning message that use of redis is deprecated?

Reply:
done, pushed a fall back